lib/uri.mli

```diff
@@ -0,0 +1,24 @@
+val get_sld : string -> string option
+(** Get the URI's second level domain, if it has one. *)
+
+val append_to_base : rel_path:string -> string -> string
+(** Append a relative path to a base URI. *)
+
+val chop_git_prefix : string -> string
+(** Chop the prefix "git+" from a URI, if there. *)
```

Comment:
I would say "if any"

Reply:
I think my English might not be good enough to understand why to say "if any" in this context. Does "if present" also sound good to you? If not, I can change it to "if any".

Reply:
Yes I think "if present" also works, maybe I'm wrong but I think "if any" is really common to say "if this thing exists"

Reply:
Ok, I've used "if any". That way I get used to it.